src/azure-cli/azure/cli/command_modules/vm/aaz/latest/vm/host/_list_host_resize_options.py

```diff
@@ -0,0 +1,184 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "vm host list-host-resize-options",
```

Comment:
```suggestion
    "vm host list-resize-options",
```

Reply:
@cakarata I suggest changing the command name from `vm host list-host-resize-options` to `vm host list-resize-options`, which is more concise and makes it easier for customers to input, what do you think of it?

Reply:
> 

I think it makes sense. I wanted to make it consistent with vm resize command but "vm host list-host-resize-options" is pure ugly.

Reply:
okay, I will rename this command.